### PR TITLE
More Tests!

### PR DESCRIPTION
**Pull request by DarthHater**
Opening a PR just so I can see the changes easier. @allenhsieh feel free to co-opt.

### CODE REVIEW

**Comment by ajurgenson55** [comment]
```
ajurgenson@ArtieSonaDell:~/git_repos/auditjs$ npm run test-ci

> auditjs@4.0.1-alpha.20 test-ci /home/ajurgenson/git_repos/auditjs
> MOCHA_FILE=./reports/test-results.xml mocha -r ts-node/register src/**/*.spec.ts --reporter mocha-junit-reporter

Sonabot here, you have some policy violations to clean up!
Report URL: 
undefined
Wonderbar! No policy violations reported for this audit!
Report URL: 
[winston] Attempt to write logs with no transports {"level":"debug","message":"api/v2/scan/applications/a20bc16e83944595a94c2e36c1cd228e/status/9cee2b6366fc4d328edc318eae46b2cb"}
[winston] Attempt to write logs with no transports {"0":{"message":"Invalid URL: api/v2/scan/applications/a20bc16e83944595a94c2e36c1cd228e/status/9cee2b6366fc4d328edc318eae46b2cb"},"level":"debug"}
[winston] Attempt to write logs with no transports {"0":{"response":{"size":0,"timeout":0}},"level":"debug","message":"response"}
```

Getting this when running npm run test-ci locally and it doesn't actually run the scan.  Is it configured to hit an IQ server on the circle ci side?  Also I would expect these tests to fail in this case.  

Coverage report is looking good though:
image

**Comment by DarthHater** [comment]
Those output lines are basically us not squelching console.log in a few of the tests ( @allenhsieh I think you know what to do!).

**Comment by allenhsieh** [comment]
> Those output lines are basically us not squelching console.log in a few of the tests ( @allenhsieh I think you know what to do!).

This got lost in my inbox, i'm going to take a look at it now and remove some console logs

**Comment by allenhsieh** [comment]
image

Log messages are now silent and ready for review.

File: .nycrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "exclude": ["**/*/*.spec.ts"]
+}
```
Comment (DarthHater) [inline .nycrc]: Trivial: newline

Comment (allenhsieh) [reply .nycrc]: in: https://github.com/sonatype-nexus-community/auditjs/pull/141/commits/47b566b5298f5f37732e706a52596faebad0333f

**Comment by allenhsieh** [comment]
thou hast been reborn in #173

**Pull request closed without merging**